### ISSUE TITLE
Allow a user to provide a commit message when saving a Document or Collection Item

### ISSUE THREAD

**Issue by revgum** [lumeland/cms#5]
When the CMS configuration is using GitHub as the storage adapter;

- It would be nice to have a text input (with a default value/placeholder) so that the user can supply a specific git commit message when saving a change.
- **Or** the default message to include the "document name" or "collection item name" (ie. "Updated ${document.name} document")

**Comment by oscarotero** [comment]
I like the second option. This is something specific for the GitHub adapter so I don't think there should be something in the UI for that. Most people won't know what to write there (or even don't know they are using Git).

The storage is independent of the collection (the same storage can be used for several collections and documents). But the default message could include, at least, the file name that is being edited/created.

If you want to make it even more customizable, we can add an option for that. For example:

```js
const gh = new GitHub({
  // ...
  commitMessage(info) {
    return `Edited file "${info.path}"`,
  }
});
```

**Comment by oscarotero** [comment]
I just added this option. Let me know if it works for you.

**Comment by revgum** [comment]
Thanks! Are you able to release version 0.3.4 with this update?

**Comment by oscarotero** [comment]
done!

**Comment by revgum** [comment]
Addressed in #8
